backend/src/server/routes/v1/project-router.ts

```diff
@@ -268,14 +268,16 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
     },
     onRequest: verifyAuth([AuthMode.JWT]),
     handler: async (req) => {
-      const { invitee, latestKey } = await server.services.projectMembership.inviteUserToProject({
+      const { invitees, latestKey } = await server.services.projectMembership.inviteUserToProject({
         actorId: req.permission.id,
         actor: req.permission.type,
         actorOrgId: req.permission.orgId,
         projectId: req.params.workspaceId,
-        email: req.body.email
+        emails: [req.body.email]
       });
 
+      const invitee = invitees[0];
```

Comment:
Are you refering to invitee or inviter? Also is this relying on the frontend to always add the first user as the one who invites everyone?